enterprise/reporting/error_reporting.go

```diff
@@ -509,10 +524,13 @@ func (edr *ErrorDetailReporter) aggregate(reports []*types.EDReportsDB) []*types
 
 		reportsCountMap := lo.CountValuesBy(reports, func(rep *types.EDReportsDB) types.EDErrorDetails {
 			return types.EDErrorDetails{
-				StatusCode:   rep.StatusCode,
-				ErrorCode:    rep.ErrorCode,
-				ErrorMessage: rep.ErrorMessage,
-				EventType:    rep.EventType,
+				StatusCode:     rep.StatusCode,
+				ErrorCode:      rep.ErrorCode,
+				ErrorMessage:   rep.ErrorMessage,
+				EventType:      rep.EventType,
+				SampleEvent:    rep.SampleEvent,
+				SampleResponse: rep.SampleResponse,
+				Count:          int(rep.Count),
```

Comment:
How does adding this `Count` serve as a fix for `count` column in errors table ?